lib/linter.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash');
+const Result = require('./result-model');
+
+class Linter {
+  constructor(options) {
+    this.matcher = options.matcher;
+    this.pathParser = options.pathParser || _.identity;
+
+    _.bindAll(this, ['filePathTest', 'lint']);
```

Comment:
Correct. Which allows usage like this:

* https://github.com/usabilla/fnlint/blob/b9b6008e2c3177d78e7d7207c9ae9339542647b8/lib/module.js#L25
* https://github.com/usabilla/fnlint/blob/b9b6008e2c3177d78e7d7207c9ae9339542647b8/lib/linter.js#L15